### PR TITLE
Generate Compute client library version 19.0.0 and update the tests.

### PR DESCRIPTION
**Pull request by hyonholee**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/_metadata/compute_resource-manager.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      58409bca13e129632c26c9d3767f22f922af62dc
+Commit:      37e2f7bf51c9a8ab6079d1cfa0f0b31b475315ff
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\hylee\AppData\Roaming\npm `-- autorest@2.0.4245  
+Bootstrapper version:    C:\Users\hylee\AppData\Roaming\npm `-- autorest@2.0.4262
```
Comment (dsgouda) [inline src/SDKs/_metadata/compute_resource-manager.txt]: Please regenerate the code using `generate.ps1` script [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1)

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part

**Comment by hyonholee** [comment]
@dsgouda   I generated the code using the powershell script and updated the PR.

**Review by dsgouda — APPROVED** [review]
LGTM subject to CI builds passing

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please fix failing tests

**Comment by hyonholee** [comment]
@dsgouda  Fixed tests and all checks passed.

**Pull request merged**